packages/storybook/stories/Stateful Client/Adapters.stories.mdx

```diff
@@ -0,0 +1,120 @@
+These adapters make use of the stateful clients for calling and chat in conjunction with React Hooks like usePropsFor to provide developers a single interface to configure Composites.
+
+## Setting Up Adapters
+
+To setup the adapters, you will need to provide the adapter with key metadata required.
+This includes most importantly an access token for your Azure Communication Services resource and context information for the call or chat thread you want to join.
+To create the adapters we will use the `createAzureCommunicationCallAdapter` and `createAzureCommunicationChatAdapter` helper methods.
+These methods are asynchronous so we will use `useEffect` to create them.
+
+### Calling Adapters
+
+To initialize the calling adapter, you will be required to provide a call locator with the information of the group or meeting you are trying to join.
+Group Id for Azure Communication Services take the shape of GUID generated by the developer to identify a call inside their tenant.
+Teams Meeting information can be acquired through [Microsoft Graph](https://docs.microsoft.com/en-us/graph/overview).
+
+<Source code={CallAdapterExampleText} />
+
+### Chat Adapter
+
+To initialize the chat adapter, you will be required to provide an endpoint url for the Azure Communication Services resource on [Azure Portal](https://portal.azure.com).
+In addition, you will need to provide a thread Id. Thread Id is provisioned from the Azure Communication Services by creating a new thread or adding the user to an existing thread.
+For more information on thread Id, see our [documentation](https://docs.microsoft.com/azure/communication-services/concepts/chat/concepts).
+
+<Source code={ChatAdapterExampleText} />
+
+## Using Adapters
+
+Once initialized, adapters can be used to initialize Composites as well as to pragmatically take actions on the call or chat experience.
+
+### Initializing Composites with Adapters
+
+With the initialized adapters, we can now hook them into the calling and chat composites.
+
+```typescript
+
+import { CallComposite, ChatComposite} from '@azure/communication-react';
+
+<CallComposite adapter={callAdapter} />
+
+<ChatComposite adapter={chatAdapter} />
+
+```
+
+### Accessing Adapter Contents
+
+The adapters provide developers access to state, actions and events related to the underlying chat and calling SDKs.
+Below are some examples that might be helpful, but additional methods are available.
+
+#### Calling Adapter Examples
+
+```typescript
+
+//Access state and state changes, including CallClient and UI
+callAdapter.onStateChange((state) => {
+  console.log(state.userId)
+})
+callAdapter.getState().userId
+callAdapter.getState().call
+callAdapter.getState().devices
+
+//Access actions for the call
+callAdapter.mute()
+callAdapter.queryMicrophones()
+callAdapter.startCamera()
+
+//Call events
+callAdapter.on('participantsJoined, (event)=>{
```

Comment:
nit: spacing here and further below
```suggestion
callAdapter.on('participantsJoined, (event) => {
```

Reply:
Maybe we can update prettier instead to run over these mdx files - looks to have some support https://prettier.io/blog/2018/11/07/1.15.0.html

Otherwise maybe an eslint plugin would help here https://www.npmjs.com/package/eslint-mdx

Reply:
I am going to create a bug for adding prettification or linting to mdx files in storybook

https://skype.visualstudio.com/SPOOL/_workitems/edit/2508320